taxonomies/labels.txt

```diff
@@ -644,7 +644,7 @@ he:נטול גלוטן
 hu:Gluténmentes
 it:Senza glutine
 nl:Glutenvrij, Gluten vrij, Vrij van gluten, Zonder gluten
-pl: Bezglutenomy
+pl: Bezglutenowy
```

Comment:
Why is there a leading space?

Reply:
The leading space will be removed, but it's better without one just for consistency.